pkg/report/report.go

```diff
@@ -108,7 +109,9 @@ func (r *pdfReporter) WriteReportPDF(reportFilename string,
 	r.createDataAssets(model)
 	r.createTrustBoundaries(model)
 	r.createSharedRuntimes(model)
-	r.createRiskRulesChecked(model, modelFilename, skipRiskRules, buildTimestamp, threagileVersion, modelHash, customRiskRules)
+	if showChapters[RiskRulesCheckedByThreagile] {
```

Comment:
Could not figure out what's the default value

Reply:
should be fixed now